leshan-client-cf/src/main/java/org/eclipse/leshan/client/californium/LeshanClientBuilder.java

```diff
@@ -122,6 +122,7 @@ public LeshanClient build() {
         }
         if (networkConfig == null) {
             networkConfig = new NetworkConfig();
+            networkConfig.set(NetworkConfig.Keys.MID_TRACKER, "NULL");
```

Comment:
It would be nice to have the enum `org.eclipse.californium.core.network.InMemoryMessageIdProvider.TrackerMode` public.